esmf_regrid/schemes.py

```diff
@@ -1,19 +1,34 @@
+# -*- coding: utf-8 -*-
 """TBD: public module docstring."""
 
+import iris
+
+from ._grid import GridInfo
+from .esmf_regridder import Regridder
+
 __all__ = [
-    "ESMFAreaWeighted",
+    "ESMF",
 ]
 
 
-class ESMFAreaWeighted:
+class ESMF:
```

Comment:
This may cause confusion since there is already a package being imported named ESMF.